{{cookiecutter.repo_name}}/src/test/kotlin/tests/Testy.kt

```diff
@@ -1,10 +1,7 @@
 package tests
 
-import org.junit.jupiter.api.Assertions.assertEquals
 import org.junit.jupiter.api.Test
 
-import frc.team{{cookiecutter.team_number}}.{{cookiecutter.repo_name}}.Robot
```

Comment:
weren't we gonna have some initial test so that it is obvious how to use tests?

Reply:
Yeah, ktlint complains about this being an unused import though. We should add it back in another PR with those example tests.

Reply:
FYI the ktlint output will look something like this when things fail:
```
ktlint wrapper...........................................................Failed
hookid: ktlint-wrapper

src/main/kotlin/frc/team5499/example_robot/Robot.kt:1:1: Package name must not contain underscore (cannot be auto-corrected) (package-name)
src/test/kotlin/tests/Testy.kt:3:1: Unused import (no-unused-imports)
src/test/kotlin/tests/Testy.kt:6:1: Unused import (no-unused-imports)
```

Reply:
Ok, for sure